ArduCopter/AP_Arming.cpp

```diff
@@ -658,6 +658,11 @@ bool AP_Arming_Copter::mandatory_checks(bool display_failure)
         result = false;
     }
 
+#if AP_OPENDRONEID_ENABLED
```

Comment:
These should just go in the parent class, rather than just in the flying vehicle.  Having this stuff around for the non-flying vehicles isn't that painful.

Reply:
fixed